src/Build/BackEnd/Components/SdkResolution/SdkResolverContext.cs

```diff
@@ -3,25 +3,26 @@
 
 using System;
 
-using SdkResolverContextBase = Microsoft.Build.Framework.SdkResolverContext;
+using SdkResolverContextBase2 = Microsoft.Build.Framework.SdkResolverContext2;
```

Comment:
This is an alias. We don't need to rename this. You can just have…

```suggestion
using SdkResolverContextBase = Microsoft.Build.Framework.SdkResolverContext2;
```

Infact, we can remove this alias entirely now that there's no conflict in naming!

Reply:
Make sense. How about very explicit
internal sealed class SdkResolverContext : Microsoft.Build.Framework.SdkResolverContext2
?

Reply:
It'll confuse readers. I personally prefer keeping the `Base` suffix, since it makes clear that the imported class is base class.